mne/stats/_adjacency.py

```diff
@@ -18,16 +19,26 @@ def combine_adjacency(*structure):
     *structure : list
         The adjacency along each dimension. Each entry can be:
 
-        - ndarray or sparse matrix
+        - ndarray or scipy.sparse.spmatrix
             A square binary adjacency matrix for the given dimension.
+            For example created by :func:`mne.channels.find_ch_adjacency`.
         - int
             The number of elements along the given dimension. A lattice
-            adjacency will be generated.
+            adjacency will be generated, which is a binary matrix
+            reflecting that element N of an array is adjacent to
+            elements at indices N - 1 and N + 1.
 
     Returns
     -------
     adjacency : scipy.sparse.coo_matrix, shape (n_features, n_features)
-        The adjacency matrix.
+        The square adjacency matrix, where the shape ``n_features``
```

Comment:
But this function **does** return a `coo_matrix` :thinking:

Reply:
Ok, if its `combine_adjacency` then it doesn't matter so much. Although it might be good to be consistent in the future. Currently `plot_ch_adjacency` allowse only dense csr sparse arrays. But this is ok, since combine_adjacency return's a matrix that shouldn't be used in plot_ch_adjacency.